### PR TITLE
Use {lo,hi} instead of {L,R} where appropriate in notes module.

### PR DESCRIPTION
**Pull request by jasondavies**
Other docs seem to use lo and hi, but at least this is self-consistent.  Might be worth picking one notation throughout all modules?

### CODE REVIEW

**Comment by hdevalence** [comment]
Hi, thanks for the PR!

Originally we were using `_L` and `_R` for the "left" and "right" halves of the vectors, because we didn't like the slice notation that the paper uses. This turned out to not be the best notation, since the distinction between the left and right sides of the inner product is important, and it's confusing to have two different senses of "left" and "right". So we changed to `_{lo}` and `_{hi}` for the low and high halves of the vectors.

All this is to say that we'd like to be consistent, but rather than changing `lo` and `hi` to `L` and `R`, it would be better to change `L` and `R` to `lo` and `hi`.

**Comment by jasondavies** [comment]
Ah, that makes sense!  If you'd like, I can update this PR to use `lo` and `hi` instead of `L` and `R` in the notes module.  Or you may prefer to do it yourself?  Either way, happy to help out.

**Comment by hdevalence** [comment]
If you'd like to make the changes yourself, feel free!

File: docs/notes.md [hunk 1]
```diff
@@ -66,13 +66,13 @@ instead of \\(\operatorname{Com}(v, {\widetilde{v}})\\) for brevity.
 We also make use of *vector Pedersen commitments*, which we define for
 pairs of vectors as \\[
 \begin{aligned}
-    \operatorname{Com}({\mathbf{a}}\_{L}, {\mathbf{a}}\_{R}) 
- &= \operatorname{Com}({\mathbf{a}}\_{L}, {\mathbf{a}}\_{R}, {\widetilde{a}})
-  = {\langle {\mathbf{a}}\_{L}, {\mathbf{G}} \rangle} + {\langle {\mathbf{a}}\_{R}, {\mathbf{H}} \rangle} + {\widetilde{a}} {\widetilde{B}},\end{aligned}
+    \operatorname{Com}({\mathbf{a}}\_{\operatorname{lo}}, {\mathbf{a}}\_{\operatorname{hi}})
```
Comment (hdevalence) [inline docs/notes.md]: These `a_L` and `a_R` are different vectors (left and right bit commitments), not the low and high halves of a single vector...

Comment (jasondavies) [reply docs/notes.md]: Ah, oops. Will fix...

File: docs/notes.md [hunk 2]
```diff
@@ -315,7 +315,7 @@ we can express the coefficients of \\(t(x)\\) using Karatsuba’s method:
 \\]
 Since \\[
 \begin{aligned}
-  t\_{0} &= {\langle {\mathbf{a}}\_{L} - z {\mathbf{1}}, {\mathbf{y}}^{n} \circ ({\mathbf{a}}\_{R} + z {\mathbf{1}}) + z^{2} 2^{n} \rangle},\end{aligned}
+  t\_{0} &= {\langle {\mathbf{a}}\_{\operatorname{lo}} - z {\mathbf{1}}, {\mathbf{y}}^{n} \circ ({\mathbf{a}}\_{\operatorname{hi}} + z {\mathbf{1}}) + z^{2} 2^{n} \rangle},\end{aligned}
```
Comment (oleganza) [inline docs/notes.md]: Here `a_L` and `a_R` are distinct vectors, not two halves of a single vector.

**Comment by oleganza** [comment]
Apart from the `L/R<->lo/hi` confusion on line 318, the rest looks good.

File: docs/notes.md (hunk 2 above)
Comment (oleganza) [reply docs/notes.md]:
```suggestion
  t\_{0} &= {\langle {\mathbf{a}}\_{L} - z {\mathbf{1}}, {\mathbf{y}}^{n} \circ ({\mathbf{a}}\_{R} + z {\mathbf{1}}) + z^{2} 2^{n} \rangle},\end{aligned}
```

**Comment by jasondavies** [comment]
Great! I've reverted the change to line 318 now.

**Comment by oleganza** [comment]
@jasondavies hey, could you pls rebase this on the latest `main` branch (505556b218d6f78f4767c38654ca5e5904b08b9e), so we can merge? I just checked, there are no conflicts, so it should be a quick thing to do.

```
git checkout main
git pull git@github.com:dalek-cryptography/bulletproofs.git main
git checkout notes
git rebase main
git push -f git@github.com:plutomonkey/ristretto-bulletproofs.git notes
```

**Comment by jasondavies** [comment]
Done!

**Comment by hdevalence** [comment]
trying to restart travis by closing and reopening this PR, since the build has gone missing

**Comment by hdevalence** [comment]
(couldn't get Travis to remember the PR's existence, but it only changes docs, so I force merged it)

**Pull request merged**